content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -75,9 +82,29 @@ Take a look at the new
 easily experiment with different parameter values.
 
 You could script a grid search pretty easily by queueing an experiment for each
-set of parameter values you want to try. For example,
-`dvc exp run --queue -S alpha={alpha},beta={beta}`, and then compare the results
-with `dvc exp show`.
+set of parameter values you want to try. For example:
+
+```dvc
+$ dvc exp run --queue -S alpha={alpha},beta={beta}
+$ dvc exp run --run-all --jobs 2
+```
+
+Then you can compare the results with `dvc exp show`.
+
+```
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
+│ master        │  0.55259 │ 0.91536 │ 50         │ 2               │
+│ ├── exp-bfe64 │  0.57833 │ 0.95555 │ 50         │ 8               │
+│ ├── exp-b8082 │  0.59806 │ 0.95287 │ 50         │ 64              │
+│ ├── exp-c7250 │  0.58876 │ 0.94524 │ 100        │ 2               │
+│ ├── exp-b9cd4 │  0.57953 │ 0.95732 │ 100        │ 8               │
+│ ├── exp-98a96 │  0.60405 │  0.9608 │ 100        │ 64              │
+│ └── exp-ad5b1 │  0.56191 │ 0.93345 │ 50         │ 2               │
+└───────────────┴──────────┴─────────┴────────────┴─────────────────┘
+```
 
 We are working on developing experiments to have features or documented patterns
 explicitly for grid search support, so definitely share any feedback to help
```

Comment:
Should we tell them where to give feedback on this?  Is this issue correct to give feedback @shcheklein ? https://github.com/iterative/dvc/issues/4283